src/com/google/javascript/jscomp/parsing/parser/Parser.java

```diff
@@ -1112,6 +1112,8 @@ private boolean peekStatementStandard() {
       case TILDE:
       case BANG:
         return true;
+      case IMPORT:
+        return peekType(1) == TokenType.OPEN_PAREN;
```

Comment:
Why is this needed?

I presume we forgot some case that should be added to ParserTest in order to test this line.
Could you add a test case there?

Reply:
I was being overly cautious. I can indeed add a test case and ensure this isn't needed.

Reply:
It wasn't needed. I removed it and added a test case to prove it wasn't needed.